jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecr/EcrUtils.kt

```diff
@@ -102,3 +222,35 @@ fun SdkRepository.toToolkitEcrRepository(): Repository? {
 
     return Repository(name, arn, uri)
 }
+
+object EcrUtils {
+    val LOG = getLogger<EcrUtils>()
```

Comment:
I'm not sure if it would be better to leave the functions as top level functions or put them in the `object`, I'm leaning towards the second because they all use `LOG`